src/index.ts

```diff
@@ -5,5 +5,6 @@ export {
     startTestServer,
     runContractTests,
     installContracts,
-    installSpecs
+    installSpecs,
+    setEnvVariable
```

Comment:
this is a different name we can use `setSpecmaticEnvironment` instead

Reply:
Added changes